administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -148,6 +155,19 @@ const ApplicationCard = ({
     return !!blueCardJuleicaEntitlement
   }
 
+  const RightElement = (): ReactElement => {
+    return (
+      <RightElementContainer>
+        {application.note && <Icon icon='annotation' intent='none' />}
```

Comment:
might be out of scope, but I feel like it could be nice to display the content of the note in the tooltip maybe?

Reply:
i thought about that. But it's out of scope. They can request that feature in the future